packages/expo/scripts/aggregate_xc_privacy.rb

```diff
@@ -0,0 +1,48 @@
+require 'plist'
+
+def aggregate_xc_privacy!(installer)
+  mapping = {}
+  project = installer.aggregate_targets.first.user_project
+  installer.pod_targets.each do |pod_target|
+    pod_target.file_accessors.each do |file_accessor|
+      file_accessor.resource_bundles.each do |bundle_name, bundle_files|
+        bundle_files.each do |file_path|
+          if File.basename(file_path) == 'PrivacyInfo.xcprivacy'
```

Comment:
Does it have to be named `PrivacyInfo`? Is this Apple's requirement/recommendation or is it just how most libraries call it?

Reply:
yes, name is required by Apple

```
By default, the file is named PrivacyInfo.xcprivacy; this is the required file name for bundled privacy manifests.
```  https://developer.apple.com/documentation/bundleresources/privacy_manifest_files